docs/.vitepress/theme/style.css

```
@@ -578,109 +572,269 @@ div[class*="language-"] > span.lang {
-  .DocSearch.DocSearch-Button {
-    width: 500px !important;
-  }
+.vp-doc .home-feature-cards .card-icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 0 0 10px 0;
+  color: #0a0a0a;
+}
+
+.vp-doc .home-feature-cards .card-icon :deep(svg) {
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":deep" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)